04_HowTos/DataReduction/DRAGONS_reduction_examples/NIRI_Imaging_Supernova/NIRI_Imaging_Supernova.ipynb

```diff
@@ -398,7 +398,9 @@
     "<a class=\"anchor\" id=\"Master_Flat\"></a>\n",
     "# Master flat field\n",
     "\n",
-    "A NIRI master flat is created from a series of lamp-on and lamp-off exposures. Each flavor is stacked, and then the lamp-off stack is subtracted from the lamp-on stack."
+    "A NIRI master flat is created from a series of lamp-on and lamp-off exposures. Each flavor is stacked, and then the lamp-off stack is subtracted from the lamp-on stack.\n",
+    "\n",
+    "The master flat will be saved as N20160102S0373_flat.fits."
```

Comment:
Change
_"The master flat will be saved as N20160102S0373\_flat.fits."_
with
_"The master flat will be saved with the suffix \_flats.fits"_